Git submodule setup note (#5568)
DataDog/dd-trace-java: CONTRIBUTING.md

git config core.hooksPath .githooks
```

## Git submodule setup

Git does not automatically update submodules when switching branches.

> Use next setting, otherwise you will need to remember to add `--recurse-submodules` to git checkout when switching to old branches.

Reviewer: ```suggestion
Add the following configuration setting or you will need to remember to add `--recurse-submodules` to `git checkout` when switching to old branches.
```

```bash
git config --local submodule.recurse true
```

> This will keep submodule `dd-java-agent/agent-jmxfetch/integrations-core` up to date.

Reviewer: ```suggestion
This will keep the submodule in `dd-java-agent/agent-jmxfetch/integrations-core` up to date.
```
